src/commands/config/ConfigDungeons.ts

```diff
@@ -909,7 +912,7 @@ export class ConfigDungeons extends BaseCommand {
                             }
                         });
                     }
-
+                    console.log("return:" + (dgnToOverrideInfo && ConfigDungeons.isDefaultOverride(cDungeon, dgnToOverrideInfo)));
```

Comment:
Use `console.info` here -- I have a tendency to delete any random `console.log`s that I find.